src/SelectField/index.js

```diff
@@ -90,11 +85,12 @@ class SelectField extends React.Component {
                 dense={dense}
             >
                 <Select
+                    name={name}
                     value={value}
                     disabled={disabled}
                     filled={filled}
                     dense={dense}
-                    onChange={this.onChange}
+                    onChange={() => onChange(name, value)}
```

Comment:
I'm not sure if the `value` of this one will be the new value? I think it'll be the old value.
But see the main comment regarding this `onChange` style.